packages/gui/src/components/nfts/NFTProfileDropdown.tsx

```diff
@@ -175,69 +175,32 @@ export default function NFTProfileDropdown(props: NFTGallerySidebarProps) {
             </ListItemIcon>
             {wallet.name} {wallet.id}
           </MenuItem>
-          {inbox && (
-            <MenuItem
-              key="inbox"
-              onClick={() => {
-                onClose();
-                handleWalletChange(inbox.id);
-              }}
-              selected={walletId === inbox.id}
-            >
-              <ListItemIcon>
-                <NFTsSmallIcon />
-              </ListItemIcon>
-              <Trans>Unassigned NFTs</Trans>
-            </MenuItem>
-          )}
-          {(remainingNFTWallets ?? []).map((wallet: Wallet) => {
-            return (
-              <MenuItem
-                key={wallet.id}
-                onClick={() => {
-                  onClose();
-                  handleWalletChange(wallet.id);
-                }}
-                selected={walletId === wallet.id}
-              >
-                <ListItemIcon>
-                  <NFTsSmallIcon />
-                </ListItemIcon>
-                {wallet.name} {wallet.id}
-              </MenuItem>
-            );
-          })}
-          {(profiles ?? []).map((profile: Profile) => (
-            <MenuItem
-              key={profile.nftWalletId}
-              onClick={() => {
-                onClose();
-                handleWalletChange(profile.nftWalletId);
-              }}
-              selected={profile.nftWalletId === walletId}
-            >
-              <ListItemIcon>
-                <PermIdentityIcon />
-              </ListItemIcon>
-              {profile.name}
-            </MenuItem>
-          ))}
-          {haveNachoNFTs && (
-            <MenuItem
-              key="nacho"
-              onClick={() => {
-                onClose();
-                handleWalletChange(-1);
-              }}
-              selected={walletId === -1}
-            >
-              <ListItemIcon>
-                <NFTsSmallIcon />
-              </ListItemIcon>
-              Nacho NFTs
-            </MenuItem>
-          )}
-        </>
+        );
+      })}
+      {(profiles ?? []).map((profile: Profile) => (
+        <MenuItem
+          key={profile.nftWalletId}
+          onClick={() => handleWalletChange(profile.nftWalletId)}
+          selected={profile.nftWalletId === walletId}
+          close
+        >
+          <ListItemIcon>
+            <PermIdentityIcon />
+          </ListItemIcon>
+          {profile.name}
+        </MenuItem>
+      ))}
+      {haveNachoNFTs && (
+        <MenuItem
+          key="nacho"
+          onClick={() => handleWalletChange(-1)}
+          selected={walletId === -1}
```

Comment:
please add close attribute

Reply:
Fixed in 84e7f5afab0b7e6c9f0c3dbba34edd413ac41912